react/BarContextProvider/index.spec.jsx

```diff
@@ -22,7 +22,7 @@ const DumbHelloWorld = translate()(({ t, f, lang }) => (
   <div>
     {t('helloworld')}
     <br />
-    {f('2020-01-06', 'DDD MMM')}
+    {f(new Date('2020-01-06'), 'd MMM')}
```

Comment:
Can't, `f` deal with this BC by itself? It it's a string, then create a new Date() by itself? In order to help the migration.

Reply:
Also, is there any codemods that are helping to convert old date notation (ex `DDD MMM`) to the new one?